tests/test_crd.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+from appgate.openapi.openapi import generate_crd, generate_api_spec, SPEC_DIR
+
+
+def test_generate_crd():
+    for version in ["v12", "v13", "v14", "v15", "v16"]:
+        open_api = generate_api_spec(Path(SPEC_DIR).parent / version)
+        for k, v in open_api.entities.items():
+            crd = generate_crd(open_api.entities[k].cls, {})
+            assert crd
```

Comment:
Could we check the actual generated schema here? At least for a few entities.

Reply:
also make sure that we respect the k8s vs appgate serialization in the schema, some fields are not supposed to be present in k8s so they dont be in the schema.

Reply:
@AitorATuin Do you remember which entity in particular I should watch out for extra fields?

Reply:
https://github.com/appgate/sdp-operator/blob/main/appgate/attrs.py#L170

See that @mandopaloooza we use the attr read_only/write_only to know if it needs to be used or not, so in your case ignore the fields read_only